core/vm/analysis.go

```diff
@@ -116,3 +116,69 @@ func codeBitmapInternal(code, bits bitvec) bitvec {
 	}
 	return bits
 }
+
+// eofCodeBitmap collects data locations in code.
+func eofCodeBitmap(code []byte) bitvec {
+	// The bitmap is 4 bytes longer than necessary, in case the code
+	// ends with a PUSH32, the algorithm will push zeroes onto the
+	// bitvector outside the bounds of the actual code.
+	bits := make(bitvec, len(code)/8+1+4)
+	return codeBitmapInternal(code, bits)
```

Comment:
It should call `eofCodeBitmapInternal` instead of `codeBitmapInternal`, shouldn't it?

Reply:
updated, thx